crates/codegen/parser/runtime/src/parser_support/mod.rs

```diff
@@ -10,16 +10,26 @@ mod separated_helper;
 mod sequence_helper;
 
 #[macro_use]
-pub mod scanner_macros;
+pub(crate) mod scanner_macros;
 
-pub use choice_helper::ChoiceHelper;
-pub use context::ParserContext;
-pub use optional_helper::OptionalHelper;
-pub use parser_function::ParserFunction;
-pub use parser_result::ParserResult;
-pub use precedence_helper::PrecedenceHelper;
-#[allow(unused_imports)] // Used when copied to shipped crate
+// These are used when copied to the product shipped crate:
+#[allow(unused_imports)]
```

Comment:
One of the N reasons I'd like us to stop copying it around :sob:

Reply:
Sigh :( 
I wonder if this can be improved with better stubs that actually import symbols similar to how product crate does it..